go/chat/utils/utils.go

```diff
@@ -1030,6 +1030,12 @@ func PresentMessageUnboxed(ctx context.Context, g *globals.Context, rawMsg chat1
 					strings.ToLower(rawMsg.GetMessageType().String())))
 			}
 		}
+		// Disable reading exploding messages until fully we release support
+		if valid.IsEphemeral() && !valid.IsEphemeralExpired(time.Now()) {
+			if ekLib := g.GetEKLib(); ekLib != nil && !ekLib.ShouldRun(ctx) {
+				return miscErr(fmt.Errorf("Unable to decrypt because current client is out of date. Please update your version of Keybase"))
```

Comment:
@mmaxim could you confirm that this looks like the right error type to return here?

Reply:
Advert exploding messags